grammar.js

```diff
@@ -102,10 +102,10 @@ module.exports = grammar({
             // $.block_comment
         ),
 
-        line_comment: $ => token(seq(
+        line_comment: $ => token(prec(-1, seq(
```

Comment:
I added this, because otherwise a literal like `"#hash"` will be parsed as a line comment.